src/colmap/sensor/bitmap.cc

```diff
@@ -35,267 +35,179 @@
 #include "colmap/util/logging.h"
 #include "colmap/util/misc.h"
 
-#include "thirdparty/VLFeat/imopv.h"
-
-#include <regex>
-#include <unordered_map>
-
-#ifdef _WIN32
-#ifndef NOMINMAX
-#define NOMINMAX
-#endif
-#include <Windows.h>
-#endif
-#include <FreeImage.h>
+#include <OpenImageIO/color.h>
+#include <OpenImageIO/imagebufalgo.h>
+#include <OpenImageIO/imageio.h>
 
 namespace colmap {
 namespace {
 
-#ifdef FREEIMAGE_LIB  // Only needed for static FreeImage.
+struct OIIOInitializer {
+  OIIOInitializer() {
+    OIIO::attribute("threads", 1);
+    OIIO::attribute("exr_threads", 1);
+  }
 
-struct FreeImageInitializer {
-  FreeImageInitializer() { FreeImage_Initialise(); }
-  ~FreeImageInitializer() { FreeImage_DeInitialise(); }
+  ~OIIOInitializer() { OIIO::shutdown(); }
```

Comment:
This seems to be available only since v2.5.3.0 https://github.com/AcademySoftwareFoundation/OpenImageIO/blob/v2.5.3.0-beta1/src/include/OpenImageIO/imageio.h#L2788-L2793

Reply:
Thank you!